autosklearn/automl.py

```diff
@@ -1631,6 +1633,94 @@ def _load_best_individual_model(self):
         )
         return ensemble
 
+    def _load_pareto_front(self) -> Sequence[VotingClassifier | VotingRegressor]:
+        if len(self._metrics) <= 1:
+            raise ValueError("Pareto front is only available for two or more metrics.")
+
+        if self._ensemble_size > 0:
+            self.ensemble_ = self._backend.load_ensemble(self._seed)
+        else:
+            self.ensemble_ = None
+
+        # If no ensemble is loaded we cannot do anything
+        if not self.ensemble_:
+            raise ValueError(
+                "Pareto front can only be accessed if an ensemble is available."
+            )
+
+        elif not isinstance(self.ensemble_, MultiObjectiveEnsembleWrapper):
+            raise ValueError(
+                "Can only retrieve a Pareto front from a multi-objective ensemble "
+                "builder."
+            )
+
+        else:
```

Comment:
This `else` isn't really needed, it would help make the code less indented

Reply:
Changed this.